src/ids-breadcrumb/ids-breadcrumb.js

```diff
@@ -0,0 +1,87 @@
+import {
+  IdsElement,
+  customElement,
+  scss,
+  mix,
+} from '../ids-base';
+
+// Import Mixins
+import { IdsEventsMixin, IdsThemeMixin } from '../ids-mixins';
+
+import styles from './ids-breadcrumb.scss';
+
+/**
+ * @type {IdsBreadcrumb}
+ * @inherits IdsElement
+ * @mixes IdsEventsMixin
+ * @mixes IdsThemeMixin
+ * @part breadcrumb
+ */
+@customElement('ids-breadcrumb')
+@scss(styles)
+class IdsBreadcrumb extends mix(IdsElement).with(IdsEventsMixin, IdsThemeMixin) {
+  constructor() {
+    super();
+  }
+
+  /**
+   * Sets the role of the children to 'listitem' for accessiblity reasons.
+   * Also bolds the last crumb.
+   */
+  connectedCallback() {
+    this.setAttribute('role', 'list');
+    for (const child of this.children) {
+      child.setAttribute('role', 'listitem');
+      if (!(child.getAttribute('font-size'))) child.setAttribute('font-size', 14);
+      child.textDecoration = 'none';
+    }
+    if (this.lastElementChild) this.lastElementChild.fontWeight = 'bolder';
+  }
+
+  /**
+   * Inner template contents
+   * @returns {string} The template
+   */
+  template() {
+    return `
+      <div class="ids-breadcrumb">
+        <nav>
+          <slot></slot>
+        </nav>
+      </div>`;
+  }
+
+  /**
+   * Appends an individual breadcrumb to the end of the stack
+   * @param {Element} breadcrumb The HTML element with which to add
```

Comment:
```suggestion
   * @param {Element} breadcrumb The HTML element with which to add
   * @private
```

And could make it #push (private class)

Reply:
Actually I intended push and pop to be public. That's how people would interface with the breadcrumbs.